service/history/taskProcessor.go

```diff
@@ -334,15 +335,17 @@ func (t *taskProcessor) initializeLoggerForTask(
 		logger = logger.WithTags(
 			tag.WorkflowID(task.WorkflowID),
 			tag.WorkflowRunID(task.RunID),
-			tag.WorkflowDomainID(task.DomainID))
-		logger.Debug("Processing transfer task")
+			tag.WorkflowDomainID(task.DomainID),
+		)
+		logger.Debug(fmt.Sprintf("Processing transfer task: %v, type: %v", task.GetTaskID(), task.GetTaskType()))
```

Comment:
do we have a task id and task type tag for logging?

Reply:
i mean the sending replication task part

Reply:
@meiliang86

Reply:
> do we have a task id and task type tag for logging?
yes, the logger is first initialized with shardID, taskID, taskType and version.. The remaining fields need to be read from the task directly.. Although I should probably add that to `queueTaskInfo` interface.